src/main/scala/org/broadinstitute/dsde/agora/server/dataaccess/permissions/PermissionsClient.scala

```diff
@@ -191,13 +190,27 @@ trait PermissionsClient {
     }
   }
 
+  private def findRemainingManagers(agoraEntity: AgoraEntity, userToRemove: String): Set[AccessControl] = {
+    listPermissions(agoraEntity).
+      filter(_.roles.canManage).
+      filterNot(_.user.equalsIgnoreCase(userToRemove)).
+      toSet
+  }
+
+  private def hasRemainingManagers(agoraEntity: AgoraEntity, userToRemove: String): Boolean = {
+    findRemainingManagers(agoraEntity, userToRemove).nonEmpty
+  }
```

Comment:
I had considered that - but left it in as a safeguard. I'm fine removing it though.